packages/storybook/src/documentation/design-tokens/lagen.md

```diff
@@ -0,0 +1,34 @@
+# Lagen van Design Tokens
+
+[design-tokens]: /docs/design-tokens-design-tokens--docs
+[esds-naamgeving]: https://medium.com/eightshapes-llc/naming-tokens-in-design-systems-9e86c7444676
+[kleurenpallet]: https://www.rijkshuisstijl.nl/publiek/modules/product/DigitalStyleGuide/default/index.aspx?ItemId=6744
+[nlds-naamgeving]: https://www.nldesignsystem.nl/handboek/design-tokens/#naamgeving
+
+Om onderscheid te kunnen maken tussen tussen verschillende onderdelen van het design zijn de [LUX design tokens][design-tokens] opgesplits in 3 verschillende lagen: `core`, `common`, en `component`. Elke laag heeft zijn eigen doel en toepassing.
+
+## Core Design Tokens
+
+De core design tokens zijn de harde waarden. Alle waarden die überhaupt mogelijk zijn om je thema te maken. Het pallet waarmee we werken. Dit is de diepste laag en heeft geen referenties naar andere lagen. Er wordt wellicht gerefereerd naar een andere core design token. Voorbeelden van design tokens in die in de `core` laag zijn gedefiniëerd bijvoorbeeld het [beschikbare kleurenpallet][kleurenpallet], de typografie van LUX, en de beschikbare witruimten.
+
+**NB**: De core design tokens zijn statisch. Dit betekent dat ze nooit van waarde zullen veranderen als het thema wijzigt of men wisselt van `light` mode naar `dark`. Het nieuwe thema zal simpelweg verwijzen naar een andere core design token.
+
+Om deze design tokens te benoemen gebruikt LUX de zelfde naamgeving als die van het [Eight Shapes Design System][esds-naamgeving], of _ESDS_.
+
+## Common Design Tokens
+
+Common design tokens beschrijven de design beslissingen die zijn genomen om een bepaald thema te maken. Bijvoorbeeld welk lettertype de koppen en tekst hebben of wat de primaire kleur van het thema is. Common design tokens verwijzen voornamelijk naar core design tokens, maar ook naar andere design tokens in de common laag.
+
```

Comment:
```suggestion
In andere systemen wordt dit ook wel de "brand" of "thema" laag genoemd.
```

Reply:
Dit heb ik bewust niet toegevoegd om verwarring te voorkomen. Wij noemen het "common" dus dat is ook hoe wij het communiceren